examples/notebooks/robust_models_1.ipynb

```diff
@@ -18,7 +18,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
Could you clear the notebook. Also need to add it to the docs yaml file that tells the docs about notebooks.

Reply:
Cleared the notebook in commit 3db279b5a842ee1a423033503d57670055a78ec0. Sorry, not sure what you mean on the docs yaml file though. This notebook was already part of statsmodels, I just added two paragraphs to it, so the reference to the notebook is already in landing.yml